src/Paket.PowerShell/System.Management.Automation.ps1

```diff
@@ -0,0 +1,2 @@
+# copies System.Management.Automation.dll from the GAC to this folder
+cp ([PSObject].Assembly.Location) .
```

Comment:
Nice trick, but it doesn't work on "most" machines (ok at least not on my machine ;-) ).

![image](https://cloud.githubusercontent.com/assets/57396/8126414/9e0e7b88-10ee-11e5-9309-36e732aa3e48.png)

can we find a way to do this directly in FAKE without calling Powershell?